views/question/evidence-list.html

```diff
@@ -0,0 +1,18 @@
+<table class="govuk-table" id="files-uploaded">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header">Uploaded files</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    {% for filename in question.evidence %}
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell"><span>{{ filename }}</span></td>
+      </tr>
+    {% else %}
+      <tr class="govuk-table__row">
+        <td>No files uploaded</td>
+      </tr>
+    {% endfor %}
+  </tbody>
+</table>
```

Comment:
I'm guessing it isn't possible to use the govukTable macro? What if we were to process the list into the correct format in the controller?

If we definitely can't use it, can you remove the include.

Reply:
Didn't appear to support HTML as a input, which will be required for the action links